certstore_windows.go

```diff
@@ -661,7 +678,10 @@ func checkStatus(s C.SECURITY_STATUS) error {
 }
 
 func (ss securityStatus) Error() string {
-	return fmt.Sprintf("SECURITY_STATUS %d", int(ss))
+	enc := make([]byte, 2, 10)
+	copy(enc, "0x")
+	h := strconv.AppendUint(enc, uint64(ss), 16)
+	return fmt.Sprintf("SECURITY_STATUS %s", string(h))
```

Comment:
I think something like this would be more idiomatic:

```suggestion
	return fmt.Sprintf("SECURITY_STATUS 0x%08X", ss)
```